InsiteTeamTask/Repositories/EventRepo/EventRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using InsiteTeamTask.MockData;
+using InsiteTeamTask.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace InsiteTeamTask.Repositories.EventRepo
+{
+    public class EventRepository : IEventRepository
+    {
+        private readonly IDataService _mockData;
+
+        public EventRepository()
+        {
+            _mockData = new MockDataService();
+        }
+
+        public Event Get(int id)
+        {
+            return _mockData.Events().Where(x => x.Id == id).FirstOrDefault();
+        }
+
+        public List<Event> GetAll()
```

Comment:
Changing this return type to an `IEnumerable` would keep things nice and generic, and means that this method doesn't have to do anything except pass between the calling method and the data service